framework/src/modules/chain/index.js

```diff
@@ -122,6 +122,9 @@ module.exports = class ChainModule extends BaseModule {
 			getNodeStatus: {
 				handler: async () => this.chain.actions.getNodeStatus(),
 			},
+			getLastBlock: {
+				handler: async () => this.chain.actions.getLastBlock(),
```

Comment:
Would it make sense to make this even public?

Reply:
I think it will need to be in 3.0, but let's not do it in this PR